src/griffe/dataclasses.py

```diff
@@ -428,6 +432,22 @@ def is_kind(self, kind: str | Kind | set[str | Kind]) -> bool:
             kind = Kind(kind)
         return self.kind is kind
 
+    @cached_property
+    def inherited_members(self) -> dict[str, Alias]:
+        """Members that are inherited from base classes."""
+        if not isinstance(self, Class):
+            return {}
+        inherited_members = {}
+        for base in reversed(self.mro()):
+            for name, member in base.members.items():
```

Comment:
I was wondering if we should iterate on *inherited* members instead of regular ones, as to recursively inflate inherited members of all classes in the chain, but that does not seem correct based on Python's MRO (C3 linearization), and that would mean a lot of back and forth (potentially exponential) between child and parent classes.